docs/docs/aztec/concepts/accounts/index.md

```diff
@@ -72,87 +63,103 @@ def entryPoint(payload):
 
-### Account contract deployment
+Since this is commonly needed when sending a [note](../concepts/storage/notes.md) to an account, we coined the term complete address for the collection of:
+- all the user's public keys,
+- [partial address](keys.md#address-keys),
+- contract address.
 
-Users will need to pay transaction fees in order to deploy their account contract. This can be done by sending Fee Juice to their account contract address (which can be derived deterministically, as mentioned above), so they have funds to pay for the deployment. Alternatively, the fee can be paid for by another account, using [fee abstraction](#fee-management).
+Once the complete address is shared with the sender, the sender can use the public keys and partial address to check that the complete address is correct, and then send the notes to that address.
 
 ### Authorizing actions
 
 Account contracts are also expected, though not required by the protocol, to implement a set of methods for authorizing actions on behalf of the user. During a transaction, a contract may call into the account contract and request the user authorization for a given action, identified by a hash. This pattern is used, for instance, for transferring tokens from an account that is not the caller.
 
-When executing a private function, this authorization is checked by requesting an _auth witness_ from the execution oracle, which is usually a signed message.
-The user's Private eXecution Environment (PXE) is responsible for storing these auth witnesses and returning them to the requesting account contract.
-Auth witnesses can belong to the current user executing the local transaction, or to another user who shared it out-of-band.
+When executing a private function, this authorization is checked by requesting an authentication witness from the execution oracle, which is usually a signed message. Authentication Witness is a scheme for authenticating actions on Aztec, so users can allow third-parties (e.g. protocols or other users) to execute an action on their behalf.
+
+The user's [Private eXecution Environment (PXE)](../concepts/pxe/index.md) is responsible for storing these auth witnesses and returning them to the requesting account contract. Auth witnesses can belong to the current user executing the local transaction, or to another user who shared it out-of-band.
 
-However, during a public function execution, it is not possible to retrieve a value from the local oracle. To support authorizations in public functions, account contracts should save in contract storage what actions have been pre-authorized by their owner.
+However, during a public function execution, it is not possible to retrieve a value from the local [oracle](../../smart_contracts/oracles/index.md). To support authorizations in public functions, account contracts should save in contract storage what actions have been pre-authorized by their owner.
```

Comment:
I believe they are now stored in a public authwit registry, not on the account contract storage